ivre/tools/ipdata.py

```diff
@@ -92,21 +94,23 @@ def my_parse_args():
                 ivre.db.db.data.ensure_indexes()
     if args.download:
         ivre.geoiputils.download_all(verbose=not args.quiet)
+    dbtofeed = {}
+    if not args.dont_feed_ipdata_cols:
+        dbtofeed["feedipdata"] = [ ivre.db.db.passive ]
+        if args.update_nmap_db:
```

Comment:
No, I do mean to have `--no-update-passive-db` and `--update-nmap-db`, because I want the default to be: "update passive DB, don't update Nmap DB". That happens quite often, for example with `./configure` scripts.